namespaceFinder.php

```diff
@@ -11,20 +11,35 @@ class namespaceFinder {
     private $wantedNs;
     private $isSafe;
 
+    /**
+     * Resolves the namespace on construction
+     * 
+     * @param string $path the namespace link
+     */
     function __construct($path){
         $this->wantedNs = $this->computeWantedNs($path);
         $this->sanitizeNs();
     }
 
     private function computeWantedNs($path){
         global $ID;
+        global $conf;
+        
+        // Convert all other separators to colons
+        if ($conf['useslash']) $path = str_replace('/', ':', $path);
+        $path = str_replace(';', ':', $path);
+
         $result = '';
         $wantedNS = trim($path);
         if($wantedNS == '') {
             $wantedNS = $this->getCurrentNamespace();
         }
         if( $this->isRelativePath($wantedNS) ) {
             $result = getNS($ID);
+            // normalize initial dots ( ..:..abc -> ..:..:abc )
+            $wantedNS = preg_replace('/^((\.+:)*)(\.+)(?=[^:\.])/', '\1\3:', $wantedNS);
+        } elseif ( $this->isPageRelativePath($wantedNS) ) {
```

Comment:
I'm afraid I don't understand in which case this would be used. In particular `$wantedNS` is supposed to represent a namespace, but if I understand correctly https://www.dokuwiki.org/namespaces the syntax with a `~` (that I was not aware of btw, thanks for drawing my attention on that part of the doc) is supposed to represent a page, so it does not sound like something nspages should support.
Long story short: I would be grateful if you can give me an example where this would make sense (having an example would also make it easier for me to write non regression tests on that feature)

Reply:
Consider the following hierarchy:
```
docs.txt
docs/process.txt
docs/process/work_process.txt
docs/process/hr_process.txt
```

if you use `<nspages ~>` in `docs:process`, it would list the pages in the `docs:process:` namespace, which are the pages `docs:process:work_process` and `docs:process:hr_process`.

Compare this to `<nspages>` which would list pages in the `docs` namespace, which is `docs:process`.

Honestly, I just wanted to help close #128.

As a side note, a typical dokuwiki user would instead have `docs/process` moved to `docs/process/start.txt` and just use `<nspages>` in `docs:process:start`.